Source/Core/LongLatGeocoderService.js

```diff
@@ -0,0 +1,54 @@
+/*global define*/
+define([
+    './Cartesian3',
+    './defaultValue',
+    '../ThirdParty/when'
+], function(
+    Cartesian3,
+    defaultValue,
+    when) {
+    'use strict';
+
+    /**
+     * Provides geocoding through Bing Maps.
+     * @alias LongLatGeocoderService
+     *
```

Comment:
Realized options aren't actually used for this service, so I just removed them instead